webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -149,7 +149,7 @@
                             </tr>
                             <tr>
                                 <th scope="row">OS</th>
-                                <td>{{ report.os_name }}</td>
+                                <td>{{ report.os_pretty_version or report.os_name }}</td>
```

Comment:
Because this becomes a python expression, what happens if `report.os_pretty_version` does not exist? E.g.
```python
>>> class F:
...   pass
...
>>> report = F()
>>> report.os_name = 'Windows NT'
>>> print report.os_pretty_name or report.os_name
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: F instance has no attribute 'os_pretty_name'
>>>
```

Jinja is pretty smart when it comes to rendering things that are not defined but whichever way we do it's better to be explicit. I know you can do something like `{% if report.os_pretty_version %}` and it becomes `False` if there is no key called `os_pretty_version` on the `report` object/dict. However, I think you get a Jinja error if you try to `{% if report.os_pretty_version %}` if `report` does not exist (i.e. `Undefined`). 

To best test this, make a test where the `os_pretty_version` simply doesn't exist. But `os_name` does. And the result should be that it prints the value of the `os_name`.